c2FmZQ/internal/crypto/passphrase.go

```diff
@@ -13,7 +13,10 @@ import (
 // Passphrase retrieves a passphrase. If cmd is set, the passphrase is the
 // output of the command. Or, if file is set, the passphrase is the content
 // of the file. Otherwise, the passphrase is read from the terminal.
-func Passphrase(cmd, file string) ([]byte, error) {
+func Passphrase(cmd, file, passphrase string) ([]byte, error) {
+	if passphrase != "" {
+		return []byte(passphrase), nil
+	}
```

Comment:
I would prefer checking cmd and file first. Could you move the new `if` below the other two?

Reply:
ok for me. done